frontend/app-development/features/overview/components/Navigation/Navigation.test.tsx

```diff
@@ -32,7 +37,9 @@ describe('Navigation', () => {
     });
 
     topBarMenuItem
-      .filter((item) => item.key !== TopBarMenu.About)
+      .filter(
+        (item) => item.key !== HeaderMenuItemKey.About && item.key !== HeaderMenuItemKey.Deploy,
```

Comment:
As in the previous comment, what was the reason for changing the filter? Another thing, this filter seems to be the same as `line 27`. Maybe we should create a filter function that does this? It looks like they have the same reason to change and can therefore be shared/DRY. 😊

Reply:
See comment above 😄